<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="plot_page_scroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>635</width>
        <height>838</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="plot_type_selection_layout">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Plot type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="plot_type_selection">
           <item>
            <property name="text">
             <string>Histogram</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KDE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ECDF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Scatterplot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lineplot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Barplot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Boxplot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pairplot (Vector)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pairplot (Raster)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Parallel coordinates (Vector)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Parallel coordinates (Raster)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="plot_parameters_container">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>250</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="standardButtons">
          <set>QDialogButtonBox::RestoreDefaults</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="plot_container">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
